### PR TITLE
Vpn tunnel guide

### PR DESCRIPTION
**Pull request by morozovalekseywot**

### CODE REVIEW

File: examples/vpnaas/base-image.tf [hunk 1]
```diff
@@ -1,9 +1,9 @@
 data "vkcs_images_image" "image" {
-  size_max    = 3 * 1024 * 1024
-  visibility  = "public"
   most_recent = true
```
Comment (ftersin) [inline examples/vpnaas/base-image.tf:2]: we do not need most_recent with default

File: templates/vpnaas/guides/vkcs_vpn_tunnel.md.tmpl [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---
+subcategory: "{{.SubCategory}}"
+layout: "vkcs"
+description: |-
+  VPN Tunnel between Sprut network and Neutron network within VKCS.
```
Comment (ftersin) [inline templates/vpnaas/guides/vkcs_vpn_tunnel.md.tmpl:5]: VPN Tunnel to local network?

File: templates/vpnaas/guides/vkcs_vpn_tunnel.md.tmpl [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+---
+
+## VPN tunnel
+
+A VPN tunnel is a network connection for securely transmitting information between two or more remote nodes on the Internet.
```
Comment (ftersin) [inline templates/vpnaas/guides/vkcs_vpn_tunnel.md.tmpl:10]: see https://cloud.vk.com/docs/en/networks/vnet/concepts/vpn

File: examples/vpnaas/base-image.tf (hunk 1 above)
Comment (morozovalekseywot) [reply examples/vpnaas/base-image.tf]: done

File: templates/vpnaas/guides/vkcs_vpn_tunnel.md.tmpl (hunk 2 above)
Comment (morozovalekseywot) [reply templates/vpnaas/guides/vkcs_vpn_tunnel.md.tmpl]: Can you explain what you want?

File: templates/vpnaas/guides/vkcs_vpn_tunnel.md.tmpl [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+Building VPN tunnel depends on VKCS SDN you are using for networking.
+Consult to `vkcs_networking_sdn` datasource to figure out available SDNs in you project.
+If both `neutron` and `sprut` are available look at `sdn` attribute of network objects you are going to plug to VPN tunnel.
+
+## Configure VPN tunnel
+
+In this guide, we configure a vpn tunnel between Neutron network and Sprut network.
+Complete example files can be viewed in the `examples/vpnaas` folder on GitHub project.
+You need both SDNs are enabled in your project to try the example.
```
Comment (schirevko) [inline templates/vpnaas/guides/vkcs_vpn_tunnel.md.tmpl:24]: > are enabled

without "are"

**Review by schirevko — CHANGES_REQUESTED** [review]

File: templates/vpnaas/guides/vkcs_vpn_tunnel.md.tmpl (hunk 4 above)
Comment (ftersin) [reply templates/vpnaas/guides/vkcs_vpn_tunnel.md.tmpl]: done

File: templates/vpnaas/guides/vkcs_vpn_tunnel.md.tmpl [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+### Neutron SDN:
+If Neutron SDN is the only SDN available in a project then a standard router (`vkcs_networking_router`)
+must be used as a private router to build VPN tunnel for Neutron networks.
+Connect them to the router and specify router ID in `router_id` argument of `vkcs_vpnaas_service` resource.
+
+Only `subnet` type is supported for `type` argument of private `vkcs_vpnaas_endpoint_group` resource.
+Neutron VPNaaS can plug several subnets to one VPN tunnel but a way it is used for that could be not supported by ceratin client VPN hardware.
```
[the remark this answers is not in the document]
Comment (ftersin) [reply templates/vpnaas/guides/vkcs_vpn_tunnel.md.tmpl]: done

File: templates/vpnaas/guides/vkcs_vpn_tunnel.md.tmpl [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+This makes VPN tunnel settings more flexible.
+
+In the guide's `examples/vpnaas/sprut_base.tf` file we prepare advanced router and Sprut networks connected to it.
+Next in `examples/vpnaas/sprut_main.tf` we build VPN tunnel.
+
+{{tffile "examples/vpnaas/sprut_main.tf"}}
+
+## Verify tunnel
+In the guide's example we start two VMs in Neutron and Sprut networks and check connectivity between them.
+See `examples/vpnaas/neutron_vm.tf`, `examples/vpnaas/sprut_vm.tf` if you are interesting in.
```
[the remark this answers is not in the document]
Comment (ftersin) [reply templates/vpnaas/guides/vkcs_vpn_tunnel.md.tmpl]: done

**Review by schirevko — APPROVED** [review]

**Pull request merged**